.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
+
+go:
+  - 1.11.x
+
+env:
+  - GO111MODULE=on
+
+install:
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $GOPATH/bin v1.12.3
+
+script:
+  - make
+  - golangci-lint run
```

Comment:
It'd be good to put this in the Makefile so you can just run `make lint`.

Reply:
Done